client/src/partials/cash/cash.js

```diff
@@ -136,6 +141,12 @@ function CashController(Cash, Cashboxes, AppCache, Currencies, Exchange, Session
   function submit(form) {
     if (form.$invalid) { return; }
 
+    // patient invoices are covered by caution
+    if (vm.slip && vm.patientBalance < 0) {
+      Notify.danger('CASH.CAUTION_COVER');
```

Comment:
This is a good idea, but I think this might need to be warning instead of an error.  The patient can have a positive balance with the hospital, but it should be up to the patient if they want to use their caution to pay a particular invoice.

What do you think?